src/components/StructuredNavigation/NavUtils/SectionHeading.js

```diff
@@ -84,24 +84,22 @@ const SectionHeading = ({
       'ramp--structured-nav__section',
       isActiveSection ? 'active' : ''
     )}
-      role="listitem" data-testid="listitem-section"
+      role='listitem' data-testid='listitem-section'
       ref={sectionRef} data-label={label}
       data-mediafrag={itemId ?? ''}>
-      <div className="section-head-buttons">
+      <div className='ramp--structured-nav__section-head-buttons'>
         <button
-          data-testid={itemId == undefined ? "listitem-section-span" : "listitem-section-button"}
+          data-testid={itemId == undefined ? 'listitem-section-span' : 'listitem-section-button'}
           ref={sectionRef} onClick={handleClick}
           className={cx(
             'ramp--structured-nav__section-title',
             !itemId && 'not-clickable'
           )}>
-          <span className="ramp--structured-nav__title"
-            aria-label={label}
-          >
-            {isRoot ? '' : `${itemIndex}. `}
-            {label}
+          <span className='ramp--structured-nav__title' aria-label={label}>
+            {isRoot ? '' : `${itemIndex}.`}
```

Comment:
Was the space after the period after the index  removed on purpose?  If so, I think it would make sense to apply the same change to ListItem and add some CSS to visually put the space back in (margin-left?).

Reply:
Yes it was intentional. 
I will make that change.